content/influxdb/v2.1/notebooks/clean-data.md

```diff
@@ -10,31 +10,90 @@ menu:
     parent: Notebooks
 ---
 
-Learn how to create a notebook that normalizes data. Walk through the following example to create a notebook that does the following:
+Learn how to create a notebook that normalizes data.
+Data normalization is the process of modifying or cleaning data to be easier to
+work with. For example, adjusting numeric values to a uniform scale,
+modifying strings, etc.
 
-- Inputs sample data from NOAA
-- Normalizes sample data with a Flux script
-- Outputs normalized data to a bucket
+Walk through the following example to create a notebook that does the following:
```

Comment:
to avoid double numbered steps and "the following" twice, we may want to pull lines 20-23 out of numbered steps into an intro clause?

Reply:
Walk through the "Normalize data with a notebook" to query NOAA..., create a Flux script cell to normalize data, and write data to a bucket.